### DISCUSSION TITLE
Can remix do ratelimit or anti spider ?

### DISCUSSION THREAD

**Discussion by qianjiachun** [remix-run/remix#4649] [Q&A] [answered]
Forgive me for asking this stupid question. Maybe this is not the responsibility of remix. So... could you please tell me?

**Comment by sergiodxa** [comment, marked as answer]
You should implement this yourself, but I would most likely not do it in Remix and instead do it in the http server (eg Express)

**Reply by qianjiachun** [reply]
ok, I understand
